bitrise.yml

```diff
@@ -404,6 +405,71 @@ workflows:
               exit 1
             fi
 
+  glob_checks:
+    description: |
+      Tests if glob matching works.
+    envs:
+    - CACHE_PATH: |
+        ~/.bitrise/**/*.yml -> ~/.bitrise/plugins/spec.yml
+        ./DoesNotExist/
+        ./DoesNotExist/ -> ./doesnotexist.yml
```

Comment:
Can you please add a test with relative path (that exists), starting with ./, starting with directory name and with a glob. I am interested in wether taking the absolute path before Globbing over it is needed at all or is the expected correct way.

Reply:
Yep